README.md

```diff
@@ -1,21 +1,36 @@
-[![Build status][travis-image]][travis-url]
-
-[travis-image]:
-https://travis-ci.org/solana-labs/solana-program-library.svg?branch=master
-[travis-url]: https://travis-ci.org/solana-labs/solana-program-library
-
 # Solana Program Library
 
 The Solana Program Library (SPL) is a collection of on-chain programs targeting
 the [Sealevel parallel
 runtime](https://medium.com/solana-labs/sealevel-parallel-processing-thousands-of-smart-contracts-d814b378192).
 These programs are tested against Solana's implementation of Sealevel,
-solana-runtime, and deployed to its mainnet.  As others implement Sealevel, we
+solana-runtime, and some are deployed to mainnet-beta.  As others implement Sealevel, we
```

Comment:
```suggestion
solana-runtime, and some are deployed to Mainnet Beta.  As others implement Sealevel, we
```

for consistency

Reply:
Sounds good. I always thought it was "mainnet-beta"!

Reply:
it is to me too!  the sentence case form had more existing instances, so i gave it the win here